flume-hdfs-sink/src/main/java/com/wipro/ats/bdre/flume/sink/hdfs/AbstractHDFSWriter.java

```diff
@@ -50,7 +50,7 @@
   private Integer numberOfCloseRetries = null;
   private long timeBetweenCloseRetries = Long.MAX_VALUE;
 
-  final static Object [] NO_ARGS = new Object []{};
+  private final static Object [] NO_ARGS = new Object []{};
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)